eternalcore-core/src/main/java/com/eternalcode/core/litecommand/handler/invalidusage/InvalidUsageConfig.java

```diff
@@ -0,0 +1,29 @@
+package com.eternalcode.core.litecommand.handler.invalidusage;
+
+import eu.okaeri.configs.OkaeriConfig;
+import eu.okaeri.configs.annotation.Comment;
+import lombok.Getter;
+import lombok.experimental.Accessors;
+
+@SuppressWarnings({"FieldMayBeFinal", "FieldCanBeLocal"})
+@Getter
+@Accessors(fluent = true)
+public class InvalidUsageConfig extends OkaeriConfig implements InvalidUsageSettings {
```

Comment:
we need to define standards regarding configs in future meeting - why are we having public fields despite having generated getters?